cloud/wfm-rest-api/src/WfmRestApi.ts

```diff
@@ -0,0 +1,51 @@
+
+import * as Promise from 'bluebird';
+import * as express from 'express';
+import * as _ from 'lodash';
+import { Db } from 'mongodb';
+import { ApiConfig, defaultConfiguration } from './ApiConfig';
+import { ApiController } from './impl/ApiController';
+import { MongoDbRepository } from './impl/MongoDbRepository';
+
+/**
+ * RESTfull api handlers for Workorders, Workflows and Results (WFM objects)
+ */
+export class WfmRestApi {
+  private config;
+  private workorderService: MongoDbRepository;
+  private workflowService: MongoDbRepository;
+  private resultService: MongoDbRepository;
+
+  constructor(userConfig?: ApiConfig) {
+    this.config = _.defaults(defaultConfiguration, userConfig);
+    this.createWFMServices();
+  }
+
+  /**
+   * Create new router for hosting WFM http api.
+   */
+  public createWFMRouter() {
+    const router: express.Router = express.Router();
+    const workorderController = new ApiController(router, this.workorderService, this.config.workorderApiName);
+    const workflowController = new ApiController(router, this.workflowService, this.config.workflowApiName);
+    const resultController = new ApiController(router, this.resultService, this.config.resultApiName);
```

Comment:
👍 That will be also easy to unit tests. Do you see api exposing each crud method or we should get entire CRUD operations as now?